tests/phpunit/unit/api/utilities/beansSanitizePath.php

```diff
@@ -67,43 +67,24 @@ public function test_should_sanitize_for_filesystem() {
 	 * Test beans_sanitize_path() should sanitize for filesystem symbolic links.
 	 */
 	public function test_should_sanitize_for_filesystem_symbolic_links() {
```

Comment:
@hellofromtonya I removed the tests for integration directory as they are completely separated now (`BEANS_TESTS_DIR` refers to the Unit directory and no longer the root).